pkg/qemu/qemu.go

```diff
@@ -466,7 +466,7 @@ func Cmdline(cfg Config) (string, []string, error) {
 			"-device", "scsi-cd,bus=scsi0.0,drive=cdrom0")
 	default:
 		// TODO: consider using virtio cdrom for all the architectures
```

Comment:
Can we use this? (See the riscv64 code above)

Reply:
Done the changes.
Verified working in intel and mac m1.

Reply:
Done the changes.
Verified in Intel and M1